src/pages/staking-hub/gno-airdrop.tsx

```
@@ -1,7 +1,6 @@
 import { useEffect, useState, useRef } from 'react';
 import styled from '@emotion/styled';
-import { useContractWrite, usePrepareContractWrite } from 'wagmi';
-import { erc20ABI, useContractRead, useWalletClient } from 'wagmi';
+import { useReadContract, useWalletClient } from 'wagmi';
```

ESLint

[error] 'useReadContract' is defined but never used.

(@typescript-eslint/no-unused-vars)